src/vs/workbench/contrib/backup/electron-sandbox/backupTracker.ts

```diff
@@ -140,8 +147,11 @@ export class NativeBackupTracker extends BackupTracker implements IWorkbenchCont
 		}
 	}
 
-	private showErrorDialog(msg: string, error?: Error): void {
-		this.dialogService.show(Severity.Error, msg, [localize('ok', 'OK')], { detail: localize('backupErrorDetails', "Try saving or reverting the dirty editors first and then try again.") });
+	private showErrorDialog(msg: string, error?: Error, fileList?: string): void {
+		const advice = localize('backupErrorDetails', "Try saving or reverting the dirty editors first and then try again.");
+		const detail = fileList ? fileList + '\n' + advice : advice;
```

Comment:
I wonder if the `details` part of the dialog should be like this:

```
Try saving or reverting the dirty editors first and then try again:
<list of editors>
```

Instead of the other way around?

Reply:
Also, is it worth including the actual error held in the 'error' variable in the dialog.  It is pushed through the log service, but, not part of the dialogue.